tests/rptest/services/redpanda.py

```diff
@@ -4511,39 +4511,58 @@ def _update_usage_stats(self, node: ClusterNode):
         if node not in self._started:
             return
 
-        def _metrics_sum(name: str) -> int:
-            samples = self.metrics_sample(name, [node])
-            if samples is None:
-                return 0
-            return int(sum(s.value for s in samples.samples))
+        metric_samples = self.metrics_samples(
+            [
+                "vectorized_io_queue_total_read_bytes_total",
```

Comment:
this could better be as a tuple (attribute, metric), like `("disk_bytes_read", "vectorized_io_queue_total_read_bytes_total")` so then the whole try below is a simple loop like `self._usage_stats[attr] = samples.get(name)`

Reply:
I have applied a pattern like this.